src/server/index.js

```diff
@@ -32,12 +39,26 @@ function processCaption(asset, client) {
     });
 }
 
+function updatePosterFrame(asset, client) {
+  const { id: videoId, customPosterFrame, posterFrameIndex } = asset.data.video;
+  const isPosterUpdated = customPosterFrame || !isNil(posterFrameIndex);
+  if (!isPosterUpdated) return;
+  return client.videos
+    .updatePosterFrame(videoId, { customPosterFrame, posterFrameIndex });
```

Comment:
Consider if this format is nicer on the eyes 😄 
```suggestion
  return client.videos.updatePosterFrame(videoId, { 
    customPosterFrame,
    posterFrameIndex
  });
```